tests/lit/lit/formats/alive2test.py

```diff
@@ -124,19 +134,26 @@ def execute(self, test, litConfig):
       return lit.Test.PASS, ''
 
     chk = self.regex_check.search(input)
-    if chk != None and output.find(chk.group(1).strip()) == -1:
+    if chk != None and output.find(chk.group(2).strip()) == -1:
       return lit.Test.FAIL, output
 
     chk_not = self.regex_check_not.search(input)
-    if chk_not != None and output.find(chk_not.group(1).strip()) != -1:
+    if chk_not != None and output.find(chk_not.group(2).strip()) != -1:
+      return lit.Test.FAIL, output
+
+    if clang_tv and exitCode != 0:
+      # clang tv should not exit with non-zero even if validation fails.
+      # Otherwise it will stop a build system such as `make`.
       return lit.Test.FAIL, output
 
     expect_err = self.regex_errs.search(input)
     if expect_err is None and xfail is None and chk is None and chk_not is None:
       # If there's no other test, correctness of the transformation should be
       # checked.
+      # In case of clang tv, it may have multiple results, so ignore validation
```

Comment:
this is kinda dangerous. why do we want to support this? Is this needed for any of the tests? if so, can you change them so this is not needed?

Reply:
issue569.cpp needed this, because one of the passes raised precondition is false error.
I modified issue569.cpp to manually check 'transformation seems to be correct' keyword to avoid the issue.